src/utils.ts

```diff
@@ -26,22 +27,33 @@ export function sendError (res: ServerResponse, error: Error | string, code?: nu
     error.statusCode || error.status ||
     500
 
-  if (debug && res.statusCode !== 404) {
+  // @ts-ignore
+  if (debug && !error.runtime && res.statusCode !== 404) {
     console.error(error) // eslint-disable-line no-console
   }
 
   // @ts-ignore
   res.statusMessage = res.statusMessage || error.statusMessage || error.statusText || 'Internal Error'
 
-  res.end(`"${res.statusMessage} (${res.statusCode})"`)
+  // @ts-ignore
+  let body = error.body || `"${res.statusMessage} (${res.statusCode})"`
```

Comment:
suggestion:
```js
  let response = { statusMessage, statusCode, body: !debug && error.body }
```

Reply:
I don't get the idea of `!debug && error.body`?

Reply:
There might be valid use cases for client to consume error body but it might potentially leak traces in production. We might do something like `(debug || !internal) ? error : 'internal error')` at least

Reply:
manipulating response structure on runtime errors is not a good idea, because applications depend on the response.  
I think this would be a good way to prevent trace leak
```js
let body: any = {
    statusCode: res.statusCode,
    statusMessage: res.statusMessage
}
if (debug || error.runtime) {
    body = error.body || body
}
``